Developer_deployment.md

```diff
@@ -38,7 +38,7 @@ This table summarizes the primary accounts you need to obtain.
 
 | Account Type      | URL                                                                                            | Notes                                                                                         |
 |-------------------|------------------------------------------------------------------------------------------------|-----------------------------------------------------------------------------------------------|
-| KBase Developer Account | [*https://accounts.kbase.us/*](https://accounts.kbase.us/)                                       | Your request will need to be approved by a person [WHO??]                                                                                              |
+| KBase Developer Account | [*https://accounts.kbase.us/*](https://accounts.kbase.us/)                                       | Your request will need to be approved by a person (email help@kbase.us if you have problems.  Non-US citizens take a long time to be approved)                                                                                              |
```

Comment:
I'd suggest wording this instead as "please note that for non-US citizens, additional verification steps are currently required that will take longer". Sounds a bit less negative :)